config/api/routes/lock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Content Lock Routes
+ */
+return [
+
+    [
+        'pattern' => '(:all)/lock',
+        'method'  => 'GET',
+        'action'  => function (string $path) {
+            return $this->parent($path)->lock(false, $this->language());
+        }
+    ],
+    [
+        'pattern' => '(:all)/lock',
+        'method'  => 'PATCH',
+        'action'  => function (string $path) {
+            return $this->parent($path)->lock($this->requestBody('user'), $this->language());
+        }
+    ],
+    [
+        'pattern' => '(:all)/lock',
+        'method'  => 'DELETE',
+        'action'  => function (string $path) {
+            return $this->parent($path)->lock(null, $this->language());
```

Comment:
Shouldn't this route check whether the lock user and the current authenticated user are the same? Removing the lock without triggering `unlock` should only be possible for the user who locked the object, no?

Also see my comment for the `src/Cms/HasLock.php` trait.